### PR TITLE
Clean lightbox

### LINKED ISSUE #632: Clean up relics of lightbox
once new show page / viewer are merged

### PR DESCRIPTION
**Pull request by hackmastera**
fixes #632

### CODE REVIEW

File: app/views/curation_concerns/file_sets/show.html.erb [hunk 1]
```diff
@@ -1,21 +0,0 @@
-<%= render 'shared/citations' %>
-
```
Comment (jrochkind) [inline app/views/curation_concerns/file_sets/show.html.erb]: intentional to get rid of `shared/citations`?  I think maybe we want that?

File: app/views/curation_concerns/file_sets/show.html.erb [hunk 2]
```diff
@@ -1,21 +0,0 @@
...
-        <%= render 'work_title', presenter: @presenter %>
-      </header>
-
-      <%# TODO: render 'show_descriptions' See https://github.com/projecthydra/sufia/issues/1481 %>
-      <%= render 'show_details' %>
-      <%= render 'users/activity_log', events: @presenter.events %>
-    </div><!-- /columns second -->
-  </div> <!-- /.row -->
-</div><!-- /.container-fluid -->
-<%= render 'lightbox_preview', presenter: @presenter, preview_id: @presenter.id %>
```
Comment (jrochkind) [inline app/views/curation_concerns/file_sets/show.html.erb]: should the `lightbox_preview` partial be removed too?

File: app/views/curation_concerns/base/_member.html.erb [hunk 3]
```diff
@@ -1,19 +0,0 @@
...
-                    "data-toggle" => "modal",
-                    title: "Preview image in lightbox" } %>
-  </td>
-  <td class="attribute filename"><%= link_to(member.link_name, contextual_path(member, @presenter)) %></td>
-  <td class="attribute date_uploaded"><%= member.try(:date_uploaded) %></td>
-  <td class="attribute permission"><%= member.permission_badge %></td>
-  <td>
-    <%= render 'actions', member: member %>
-  </td>
-</tr>
```
Comment (jrochkind) [inline app/views/curation_concerns/base/_member.html.erb]: weird, I don't get how this wasn't causing a problem being there!  Maybe we weren't using that partial anymore. Cool.

File: app/views/curation_concerns/file_sets/show.html.erb (hunk 1 above)
Comment (jrochkind) [inline app/views/curation_concerns/file_sets/show.html.erb]: oh, the whole partial is going away! okay!

File: app/views/curation_concerns/file_sets/show.html.erb (hunk 2 above)
Comment (hackmastera) [inline app/views/curation_concerns/file_sets/show.html.erb]: Yes, good point; done!

**Pull request merged**